src/components/Timesheet/index.tsx

```diff
@@ -64,21 +45,87 @@ const Timesheet = () => {
         });
     }, []);
 
+    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setTimeTracked(() => parseInt(e.target.value));
+    };
+
     return (
-        <table>
-            <thead>
-                <tr>
-                    <td>Monday</td>
-                    <td>Tuesday</td>
-                    <td>Wednesday</td>
-                    <td>Thursday</td>
-                    <td>Friday</td>
-                    <td>Saturday</td>
-                    <td>Sunday</td>
-                </tr>
-            </thead>
-            <tbody></tbody>
-        </table>
+        <div className={styles.timesheet}>
+            <table className={styles.table}>
+                <thead>
+                    <tr>
+                        {days.map((day) => (
+                            <td key={day}>
+                                <div className={styles.days}>{day}</div>
+                            </td>
+                        ))}
+                    </tr>
+                </thead>
+
+                <tbody>
+                    {dates.length > 0 &&
+                        dates.map((week) => (
+                            <tr
+                                key={JSON.stringify(week[0])}
+                                className={styles.row}
+                            >
+                                {week.map(
+                                    (each: {
+                                        date:
+                                            | string
+                                            | React.ReactChild
+                                            | React.ReactFragment
+                                            | React.ReactPortal
+                                            | null
+                                            | undefined;
+                                        month: string;
+                                    }) => (
```

Comment:
I have a couple of suggestions here
1. Can we rename each to `day`?
2. instead of listing null and undefined as possible value types, you can do this:
`date?:` 
3. Please extract it to a separate type instead of listing possible types inline
```
type TimesheetDate {
     date?: string | React.ReactNode;
     month: string;
}
```
ReactNode is a built in type which can replace the ones you listed (React.ReactChild, React.ReactFragment, React.ReactPortal)
![image](https://user-images.githubusercontent.com/16070688/147917947-cb0926cc-70a5-487f-8ae2-625787c525ab.png)

and then use `date: TimesheetDate`

Reply:
Good point. I will change this. 
Thanks for suggestion .